stylus/components/button.styl

```diff
@@ -20,19 +21,19 @@ secondaryTheme = {
     contrastColor: var(--secondaryButtonContrastColor)
 }
 ghostTheme = {
-    primaryColor: var(--zircon), secondaryColor: var(--frenchPass), activeColor: var(--frenchPass), contrastColor: var(--primaryColor)
+    primaryColor: var(--zircon), secondaryColor: palette['frenchPass'], activeColor: palette['frenchPass'], contrastColor: var(--primaryColor)
 }
 highlightTheme = {
-    primaryColor: var(--emerald), secondaryColor: var(--emerald), activeColor: var(--malachite), contrastColor: var(--white)
+    primaryColor: palette['emerald'], secondaryColor: palette['emerald'], activeColor: palette['malachite'], contrastColor: var(--white)
 }
 dangerTheme = {
-    primaryColor: var(--pomegranate), secondaryColor: var(--pomegranate), activeColor: var(--monza), contrastColor: var(--white)
+    primaryColor: palette['pomegranate'], secondaryColor: palette['pomegranate'], activeColor: var(--monza), contrastColor: var(--white)
```

Comment:
I don't understand why we have a mix of CSS variables and palette here. I think we should use CSS variables only, to allow custom themes. But if it is not possible currently, we should use only palette colors to avoid having surprising things on custom themes.

Reply:
The palette colors are used for CSS variables that are not themable, this is why I use them.